Web_Development/FrontEnd/HTML_Commands/HML_Commands.md

```diff
@@ -1 +1,83 @@
-hi
\ No newline at end of file
+### The html tag: 
+The <html> tag points that the code is written in html and </html> tag states that it's the end of html code.
+
+### The head tag: 
+The head of an html code contains special information, for example: it's title. 
+
+```html 
+<html>
+   <head>
+      <title>Hello! Welcome to Girlscript Winter of Contributing</title>
+   </head>
+</html>
+```
```

Comment:
It doesn't only contain one thing, under the head there is so much important info, please mention that too.